core/computed/viewport-meta.js

```diff
@@ -35,7 +35,16 @@ class ViewportMeta {
     }
 
     const viewportProps = parsedProps.validProperties;
-    const isMobileOptimized = Boolean(viewportProps.width || viewportProps['initial-scale']);
+    const initialScale = viewportProps['initial-scale'];
+    if (initialScale && initialScale < 1) {
```

Comment:
I don't think we need an extra `if` case here. WDYT about adding this condition to the `isMobileOptimized` calculation below.